hexrd/ui/mask_manager_dialog.py

```diff
@@ -250,6 +239,8 @@ def write_all_masks(self, h5py_group=None):
             for i, (det, mask) in enumerate(data):
                 parent = d.setdefault(det, {})
                 parent.setdefault(name, {})[str(i)] = mask
+        if self.threshold:
+            d['threshold'] = HexrdConfig()._threshold_data
```

Comment:
I tried writing out just the masks (via "Export Masks"), and sometimes, this 'threshold' group in the HDF5 file is just an empty group. This produces an error when we load in the masks.

Maybe there are some situations where `HexrdConfig()._threshold_data` is invalid?

But I can't actually reproduce this issue now. Let's keep an eye out for it.